packages/web/src/scss/components/Pagination/_Pagination.scss

```diff
@@ -2,7 +2,7 @@
 @use '../../settings/cursors';
 
 .Pagination {
-    display: flex;
+    display: inline-flex;
```

Comment:
I think I was trying to implement this as a "visionary" behaviour that might be helpful for someone when I implemented `Pagination` component here https://github.com/lmc-eu/spirit-design-system/pull/498#pullrequestreview-1125136201 but @crishpeen didn't like it and at the moment it seemed unnecessary.

I'm just pulling it up because if I'm not mistaken and you're trying to do the same thing, my solution was shorter and was working as well.

Please have a look.

Reply:
Did I? It could be, it is already two months, so I don't remember exactly. (but I don't see it in the comments either) 

Anyway, you are correct that we didn't know we needed that earlier. 

The question is if it should be a variant. But we would have to check this with the designers.

Reply:
I was thinking about the layout variant too, but I chose quicker solution. 🙂 

Anyway, I will discuss about this topic with Michal and let you know. Thank you for comment!

Reply:
We're stick with this solution and let the parent element solve the centering. 
According design methodology, those types of component shouldn't solve the alignment.